tools/ci/website_build.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -x
+
+neutral_status=78
+source_revision=$(git rev-parse HEAD)
+
+function modifies_relevant_files {
+  base_revision=$(
```

Comment:
I think we should roll the "check if the branch is master" part into this, to considerably cut the overhead (i.e. just one action rather than two).

Reply:
Sure thing